packages/web-react/src/components/Accordion/README.md

```diff
@@ -69,20 +69,108 @@ const [openState, setOpenState] = useState<AccordionOpenStateType>(['AccordionIt
+    <AccordionHeader>Accordion Header</AccordionHeader>
+    <AccordionContent>Accordion Content</AccordionContent>
+  </AccordionItem>
+  <AccordionItem id="AccordionItemExample1">
+    <AccordionHeader>Accordion Header</AccordionHeader>
+    <AccordionContent>Accordion Content</AccordionContent>
+  </AccordionItem>
+  ...
+</UncontrolledAccordion>
+```
+
+### Uncontrolled Accordion with opened on init and default open value
+
+```javascript
+<UncontrolledAccordion defaultOpen={['AccordionItemExample1']} stayOpen>
+  <AccordionItem id="AccordionItemExample0">
+    <AccordionHeader>Accordion Header</AccordionHeader>
+    <AccordionContent>Accordion Content</AccordionContent>
+  </AccordionItem>
+  <AccordionItem id="AccordionItemExample1">
+    <AccordionHeader>Accordion Header</AccordionHeader>
+    <AccordionContent>Accordion Content</AccordionContent>
+  </AccordionItem>
+  ...
+</UncontrolledAccordion>
+```
+
 ## Accordion Props
 
 | Prop name          | Type                          | Default     | Required | Description                                      |
 | ------------------ | ----------------------------- | ----------- | -------- | ------------------------------------------------ |
 | `id`               | `string`                      | -           | no       | Accordion id                                     |
 | `children`         | `ReactNode`                   | -           | yes      | Accordion children node                          |
 | `elementType`      | `'section', 'article', 'div'` | `'section'` | no       | Type of element used as wrapper                  |
-| `open`             | `string, string[]`            | `[]`        | no       | Open state \*                                    |
+| `open`             | `string, string[]`            | -           | no       | Open state \*                                    |
 | `toggle`           | `(id: string) => void`        | -           | no       | A generic handler for a single **AccordionItem** |
 | `UNSAFE_className` | `string`                      | -           | no       | Wrapper custom class name                        |
 | `UNSAFE_style`     | `CSSProperties`               | -           | no       | Wrapper custom style                             |
 
 (\*) Depending on the type of default value, what is set as the default will affect whether one or more will be open at the same time.
 
+## Uncontrolled Accordion Props
+
+| Prop name          | Type                          | Default     | Required | Description                                     |
+| ------------------ | ----------------------------- | ----------- | -------- | ----------------------------------------------- |
+| `id`               | `string`                      | -           | no       | Accordion id                                    |
+| `children`         | `ReactNode`                   | -           | yes      | Accordion children node                         |
```

Comment:
Since children stands for multiple nodes because it is plural, I would prefer "Accordion children's nodes" or "Accordion child nodes"

Reply:
done